### PR TITLE
[res] CircleRecipes and TFLiteRecipes added for RmsNorm

### PR DESCRIPTION
**Pull request by seockho-kim**
This commit adds RmsNorm test recipe in CircleRecipes/TFLiteRecipes.

ONE-DCO-1.0-Signed-off-by: Seockho Kim seockho.kim@samsung.com

issue: https://github.com/Samsung/ONE/issues/13964
draft: https://github.com/Samsung/ONE/pull/13967

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.rule [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# To check if this network is converted to circle RmsNorm op
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "RMS_NORM_EXIST"     $(op_count RMS_NORM) '=' 1
+RULE    "NO_ADD"                  $(op_count ADD) '=' 0
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.rule:6]:
```suggestion
RULE    "RMS_NORM_EXIST"          $(op_count RMS_NORM) '=' 1
RULE    "NO_ADD"                  $(op_count ADD) '=' 0
```

File: res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe [hunk 2]
```diff
@@ -0,0 +1,125 @@
+operand {
+  name: "Input"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1
+    dim: 1440
+  }
+  is_variable: false
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe:9]: I think `is_variable` can be removed.

Comment (seockho-kim) [reply res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe:9]: Okay, I'll update it. :)

File: res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.rule (hunk 1 above)
Comment (seockho-kim) [reply res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.rule:6]: ^^;; 
I'll update it.

**Comment by seanshpark** [comment]
> [ 76%] Generate RmsNorm_000.opt.circle
terminate called after throwing an instance of 'oops::UserExn'

check with adding to `exclude.lst` like
```
## CircleRecipes

optimize(RmsNorm_000)
```

**Comment by seockho-kim** [comment]
> > [ 76%] Generate RmsNorm_000.opt.circle
> > terminate called after throwing an instance of 'oops::UserExn'
> 
> check with adding to `exclude.lst` like
> 
> ```
> ## CircleRecipes
> 
> optimize(RmsNorm_000)
> ```

Thanks :) 
I've tried to fix it, but I couldn't
I'll update it

File: compiler/common-artifacts/exclude.lst [hunk 3]
```diff
@@ -9,6 +9,7 @@ optimize(Add_STR_000) # STRING is not supported
 optimize(Add_STR_001) # STRING is not supported
 
 ## CircleRecipes
+optimize(RmsNorm_000)
```
Comment (seanshpark) [inline compiler/common-artifacts/exclude.lst:12]: plz split this to another PR.

Comment (seockho-kim) [reply compiler/common-artifacts/exclude.lst:12]: > plz split this to another PR.

Okay, I'll submit another PR.

Comment (seockho-kim) [reply compiler/common-artifacts/exclude.lst]: It's merged as https://github.com/Samsung/ONE/pull/14032

File: res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe [hunk 4]
```diff
@@ -0,0 +1,117 @@
+operand {
+  name: "Input"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1
+    dim: 1440
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe:7]: For faster execution in luci-interpreter, can we have smaller single digit value instead of `1440`?

Comment (seockho-kim) [reply res/TensorFlowLiteRecipes/Net_RmsNorm_000/test.recipe:7]: Okay, I'll change 1440 -> 4.

**Review by seanshpark — APPROVED** [review]
LGTM thank you!
